File: Backend/src/main/i5/las2peer/services/immersiveProjectManagementService/i5/las2peer/services/immersiveProjectManagementService/dataModel/requirementsBazaar/Project.java
package i5.las2peer.services.immersiveProjectManagementService.i5.las2peer.services.immersiveProjectManagementService.dataModel.requirementsBazaar;

/**
 * Created by bened on 10.05.2019.
 */
//@JsonFilter("shortFilter")
public class Project {
    public int id;
    public String name;
    public String description;
    public boolean visibility;
    public int defaultCategoryId;
    public ReqBazUser leader;
    public String creationDate;
    public String lastUpdatedDate;
    public int numberOfCategories;
    public int numberOfRequirements;
    public int numberOfFollowers;
    public boolean isFollower;
}

File: Frontend/VIAProMa/Assets/Scripts/DataModel/RequirementsBazaar/Project.cs
using System;

namespace i5.VIAProMa.DataModel.ReqBaz
{
    /// <summary>
    /// Represents a project in the Requirements Bazaar
    /// </summary>
    [Serializable]
    public class Project
    {
        /// <summary>
        /// The values of a project
        /// </summary>
        public int id;
        public string name;
        public string description;
        public bool visibility;
        public int defaultCategoryId;
        public ReqBazUser leader;
        public string creationDate;
        public string lastUpdatedDate;
        public int numberOfCategories;
        public int numberOfRequirements;
        public int numberOfFollowers;
        public bool isFollower;
    }
}